catalyst_voices/Earthfile

```diff
@@ -18,13 +18,30 @@ builder:
     COPY +repo-catalyst-voices/repo .
     DO flutter-ci+BOOTSTRAP
 
+# Creates filtered OpenAPI spec
+# Takes json file from openapi-filter from /packages/internal/catalyst_voices_services
+filter-openapi:
+    FROM node:18
+    WORKDIR /packages/internal/catalyst_voices_services
+
+    COPY catalyst-gateway+build/doc/cat-gateway-api.json openapi/cat-gateway-api.json
+    COPY packages/internal/catalyst_voices_services/openapi-filters.json openapi-filters.json
+
+    RUN npm install -g openapi-format
+    RUN openapi-format openapi/cat-gateway-api.json -o openapi/filtered-openapi.json --filterFile openapi-filters.json --verbose
+
+    RUN rm openapi/cat-gateway-api.json
+
+    SAVE ARTIFACT openapi/filtered-openapi.json 
+
 # Generates flutter code.
 # Generates codes for Catalyst Gateway OpenAPI, Voices Localization and
 # VoicesAssets and other packages that depend on code-generator.
 # It accepts [save_locally] ARG that when true place the artifacts in the
 # proper folders 
 code-generator:
-    ARG save_locally=false
+    ARG save_locally=false 
+    ARG use_filtered_openapi=true
```

Comment:
how about calling it `filter_openapi` instead? Please also add a single line of documentation similar to the one for save_locally that explains what this argument does